conda.recipe/run_test.py

```diff
@@ -0,0 +1 @@
+import datashader; datashader.test()
```

Comment:
What's the argument for installing the tests as part of the package, instead of how we do it in dask? https://github.com/blaze/dask/blob/master/conda.recipe/run_test.py